core-api/src/controllers/auth.ts

```diff
@@ -0,0 +1,47 @@
+import { Request, Response } from "express";
+import { AuthService } from "../services/auth";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import Messages from "../utils/messages";
+
+export class AuthController {
```

Comment:
It's important when we are creating a lot of files with the same name (auth.ts) add a middle-extension. Imagine that we trying access this file by the name, will appear a lot of files and you probably will waste a lot of time searching for this specific one. Please, rename it for auth.controller.ts.

Reply:
Please, include the middle-extension in the other files too.